app/views/local_transaction/unavailable_service.html.erb

```diff
@@ -8,27 +8,24 @@
       We've matched the postcode to <span class="local-authority"><%= @local_authority.name %></span>.
     </p>
 
-  <% if @service.body.present? %>
-    <p class="govuk-body"><%= @service.body %></p>
-  <% end %>
-
-    <p id="get-started" class="get-started group">
-      <%= render "govuk_publishing_components/components/button", {
-        text: @service.button_text,
-        rel: "external",
-        margin_bottom: true,
-        href: @service.button_link,
-      } %>
+    <p class="govuk-body">
+      <%= "This service is not available in #{@service.country_name}." %>
     </p>
-  </div>
 
-  <% if @publication.more_information.present? %>
-    <section class="more">
-      <div class="more">
-        <%= render "govuk_publishing_components/components/govspeak", {} do %>
-          <%= raw @publication.more_information %>
-        <% end %>
-      </div>
-    </section>
-  <% end %>
```

Comment:
This seems good to remove - it's seems optimistic that this will always complement the unavailable copy, and it will have just been on the previous page.

Reply:
I have added this to the unavailable page. Do you think we would need it for the devolved administration page? I thought not at first as I am worried that England related content will display there. However there is additional info in the current YML file. It feels safer to leave it out...

Reply:
Hmm yeah it's tricky to work out what's best. I don't think it should be on unavailable but not on devolved_administration_service - as I imagine any arguments that it should be on unavailable also apply to devolved_administration_service. 

I'd say either have it on both unavailable and devolved_administration_service or neither, I don't mind too much which both have pro's and con's so go with which one you're most comfortable with.

Reply:
I removed them both for now. If design/content people decide that they want it, then it will be easy enough to add it in again.

Reply:
Great stuff 👍 I think the last thing to do is to document that decision with the commit message.

Reply:
👍